From 180b51120999db622642d084c0fa7748652bf6f6 Mon Sep 17 00:00:00 2001
From: Bijeesh B <quic_bijeb@quicinc.com>
Date: Fri, 7 Mar 2025 19:43:32 +0530
Subject: [PATCH] Fix for resolving compilation issue of libswla

Fix to resolve compilation issue of libswla package of prpl.

Signed-off-by: Bijeesh B <quic_bijeb@quicinc.com>
---
 libs/libswla/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libs/libswla/Makefile b/libs/libswla/Makefile
index 9dff543..c5810e4 100644
--- a/libs/libswla/Makefile
+++ b/libs/libswla/Makefile
@@ -18,6 +18,8 @@ PKG_BUILD_DEPENDS += doxygen/host
 
 include $(INCLUDE_DIR)/package.mk
 
+TARGET_CFLAGS += -Wno-error=format-truncation
+
 define Package/$(PKG_NAME)
   CATEGORY:=prpl Foundation
   SUBMENU:=Libraries
-- 
2.17.1

